test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 587400
+//   codeDepositCost: 588800
```

Comment:
It is weird - I checked the optimized yul code (from the new codegen) and it actually looks shorter to me. So the increase seems to be in the code emitted by the old codegen.